### PR TITLE
update projects and lab descriptions and added new project

### PR DESCRIPTION
**Pull request by p3t3r67x0**

### CODE REVIEW

File: content/labs/flensburg.md [hunk 1]
```diff
@@ -37,4 +46,4 @@ leads:
   url: mailto:roald@grain.one
 ---
 
-Wir sind eine Gruppe von ehrenamtlich Aktiven, die sich für Offene Daten, Freie und Offene Software und Open Government interessiert. Unser OK Lab setzt sich lokal für eine Verbesserung der Transparenz von Staat und Verwaltungen ein, stößt Projekte zur Bürgerbeteiligung an und sucht den Austausch mit den Verwaltungen, um gemeinsam Lösungen für die Herausforderungen in unserer Region zu finden. Wir treffen uns regelmäßig, um unsere Projekte weiterzuentwickeln.
+Wir sind eine Gruppe von ehrenamtlich Aktiven, mit Expertise in Datenverarbeitung, UI/UX Design, Softwareentwicklung, Stadtplanung, Verwaltung und Datenjournalismus. Wir setzen engagieren uns für mehr Transparenz von Behördendaten. Wir machen aus offen Daten, nutzerfreundliche Visualisierungen, Geo Informations Anwendungen und Hardware Projekte zur Messung von Umweltdaten. Wir treffen uns regelmäßig und lernen von einander und arbeiten an den Projekten. Wir erstellen gemeinsam Anwendungen aus offenen Behördendaten mit dem Ziel mehr Beteiligung zu fördern und Transparenz in Verwaltungen zu etablieren.
```
Comment (joergreichert) [inline content/labs/flensburg.md:49]:
```suggestion
Wir sind eine Gruppe von ehrenamtlich Aktiven mit Expertise in Datenverarbeitung, UI/UX Design, Softwareentwicklung, Stadtplanung, Verwaltung und Datenjournalismus. Wir engagieren uns für mehr Transparenz von Behördendaten. Wir machen aus offen Daten nutzerfreundliche Visualisierungen, Geoinformations-Anwendungen und Hardware-Projekte zur Messung von Umweltdaten. Wir treffen uns regelmäßig und lernen von einander und arbeiten an den Projekten. Wir erstellen gemeinsam Anwendungen aus offenen Behördendaten mit dem Ziel mehr Beteiligung zu fördern und Transparenz in Verwaltungen zu etablieren.
```

File: content/projekte/2023-04-29-fl-open-trees-map.md [hunk 2]
```diff
@@ -1,41 +1,43 @@
...
+
+- name: Gabi Ritter
 
 tags:
 - Umwelt
 
 ---
 
-Im ersten Schritt haben wir einen vorhandenen Datensatz des Baumkatasters der Stadt Gelsenkirchen genutzt, um eine einfache Darstellung auf der Karte für den Dialog mit der Stadt und dem TBZ Flensburg zu erleichtern. Am Wasser nehmen wir besonders schnell klimatische Wetterveränderungen wahr. Wir haben uns daher mit folgenden Fragen beschäftigt:
+Im ersten Schritt haben wir einen vorhandenen Daten der Straßenbäume der Stadt Flensburg genutzt, um eine einfache Darstellung auf der Karte als Transparente Anlaufstelle für Interesierte zu schaffen. Hier oben im Norden machen sich klimatische Wetterveränderungen besonders bemerkbar. Deshalb stellen wir uns diefolgenden Fragen:
```
Comment (joergreichert) [inline content/projekte/2023-04-29-fl-open-trees-map.md:34]:
```suggestion
Im ersten Schritt haben wir einen vorhandenen Datensatz der Straßenbäume der Stadt Flensburg genutzt, um eine einfache Darstellung auf der Karte als transparente Anlaufstelle für Interessierte zu schaffen. Hier oben im Norden machen sich klimatische Wetterveränderungen besonders bemerkbar. Deshalb stellen wir uns die folgenden Fragen:
```

File: content/projekte/2023-08-08-fl-open-surface-map.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- name: Alexaner Eichhorn
+
+- name: Steffen Kunst
+
+tags:
+- Umwelt
+
+---
+
+Diese interaktive Karte zeigt die Verteilung der verschiedenen Flächennutzungsarten beim klick auf die entsprechende Gemeinde an. So lässt sich in Kürze herausfinden in welchen Regionen der Waldflächenanteil besonders hoch ist. Dies ist ein erster Prototyp, welcher mit mehr Filter Möglichkeiten ausgebaut werden soll. Zudem wollen wir auch die Daten der kreisfreien Städte und Stadtstaaten mit aufnehmen.
```
Comment (joergreichert) [inline content/projekte/2023-08-08-fl-open-surface-map.md:30]:
```suggestion
Diese interaktive Karte zeigt die Verteilung der verschiedenen Flächennutzungsarten beim Klick auf die entsprechende Gemeinde an. So lässt sich in Kürze herausfinden in welchen Regionen der Waldflächenanteil besonders hoch ist. Dies ist ein erster Prototyp, welcher mit mehr Filter-Möglichkeiten ausgebaut werden soll. Zudem wollen wir auch die Daten der kreisfreien Städte und Stadtstaaten mit aufnehmen.
```

File: static/projects/flensburg/screenshot_social_map.png
Comment (joergreichert) [inline static/projects/flensburg/screenshot_social_map.png:1]: maybe try to get file size also around 150 KB like the baumkataster_stadt_flensburg.png ?

File: static/projects/flensburg/screenshot_surface_map.png
Comment (joergreichert) [inline static/projects/flensburg/screenshot_surface_map.png:1]: maybe try to get file size also around 150 KB like the baumkataster_stadt_flensburg.png ?

File: static/projects/flensburg/spielplaetze_in_flensburg.png
Comment (joergreichert) [inline static/projects/flensburg/spielplaetze_in_flensburg.png:1]: maybe try to get file size also around 150 KB like the baumkataster_stadt_flensburg.png ?

File: content/projekte/2023-04-29-fl-open-trees-map.md [hunk 4]
```diff
@@ -1,41 +1,43 @@
 ---
 layout: project
 lab: [flensburg]
-imgname: flensburg/baumkataster_gelsenkirchen.png
-title: Baumkataster Karte
+imgname: flensburg/baumkataster_stadt_flensburg.png
+title: Straßebäume der Stadt Flensburg
```
Comment (joergreichert) [inline content/projekte/2023-04-29-fl-open-trees-map.md:5]:
```suggestion
title: Straßenbäume der Stadt Flensburg
```

**Review by joergreichert — CHANGES_REQUESTED** [review]
see my comments inline

**Comment by digital-codes** [comment]
What about importing lab projects?

**Comment by p3t3r67x0** [comment]
@joergreichert thank you for your kind review, just commited suggested changes and reduced file size from screenshots
@digital-codes just added flag to showcase lab projects in overview
@all happy to get merged or feedback

**Review by joergreichert — APPROVED** [review]

**Pull request merged**